config/locales/titles/en.yml

```diff
@@ -10,9 +10,10 @@ en:
       password: Edit your password
       phone: Edit your phone number
     enter_2fa_code: Enter the secure one-time security code
-    loa3_verified:
-      'true': You have verified your identity with %{app}
-      'false': You have created your account with %{app}
+    loa3_verified_html: You have %{accent} with %{app}
+    loa3_verified_accent:
```

Comment:
Would it just be easier to have two separate strings?

```
loa3_created_html:
loa3_verified_html: 
```
Since `true` and `false` are meaningless as keys IMO